packages/frontend-2/pages/dashboard.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="flex flex-col gap-y-12">
+    <section>
+      <h2 class="text-heading-sm text-foreground-2">Quick start</h2>
+      <div class="flex flex-col md:grid md:grid-cols-3 gap-3 pt-5">
+        <QuickStartCard
+          v-for="(quickStartItem, index) in quickStartItems"
+          :key="`quick-start-${index}`"
+          :title="quickStartItem.title"
+          :description="quickStartItem.description"
+          :buttons="quickStartItem.buttons"
+        />
+      </div>
+    </section>
+    <section>
+      <h2 class="text-heading-sm text-foreground-2">Recently updated projects</h2>
+      <div class="flex flex-col md:grid md:grid-cols-3 gap-3 pt-5">
+        <DashboardProjectCard
+          v-for="project in projects"
+          :key="project.id"
+          :project="project"
+        />
+      </div>
+    </section>
+    <section>
+      <h2 class="text-heading-sm text-foreground-2">News &amp; tutorials</h2>
```

Comment:
Later this will be just tutorials, but now the feedback is a mix, we can update this text when we have tutorials only feed but not such a prio I think